src/data.js

```diff
@@ -106,23 +56,88 @@ export const weitereInfos = {
     link: "https://wir-bleiben-liqui.de/tag/buergschaft/"
   },
   Kurzarbeit: {
-    text: "<p>Das Unternehmen zahlt anteiligen Lohn basierend auf tatsächlich gearbeiteten Stunden. Der Lohn wird durch die Bundesagentur für Arbeit mit 60 % (für MA mit Kind 67 %) des ausgefallenen pauschalierten Nettoentgelts aufgestockt.</p><p>Die Regel-Bezugsdauer beträgt maximal 12 Monate. Eine Unterbrechung von bis zu 3 Monaten bei Wiederaufnahme des Geschäfts ist möglich.</p><p>Kurzarbeitergeld kann über das Portal der Bundesagentur für Arbeit beantragt werden:</p><ul><li>Antragstellung an die zuständige Agentur für Arbeit</li><li>Prüfung der Daten und Entscheidung</li><li>Kurzarbeitergeld kann bereits rückwirkend ab 01.03.2020 beantragt werden</li></ul><p>Das Unternehmen trägt die bisherigen Sozialversicherungsbeiträge zu 80 %. Dazu ist allerdings eine Gesetzesänderung geplant</p><p>Wichtig: Das Kurzarbeitergeld wird erst nach Zahlung des Lohns/Gehalts durch das Unternehmen erstattet.</p>",
     link: "https://www.arbeitsagentur.de/unternehmen/finanziell/kurzarbeitergeld-bei-entgeltausfall"
   },
   "Sozialbeiträge": {
-    text: "<p>Eine Stundung der Sozialversicherungsbeiträge kann erfolgen, wenn ein Unternehmen aufgrundder aktuellen Krise in erhebliche finanzielle Schwierigkeiten gerät.</p><p>Voraussetzung ist eine erhebliche Härte für das Unternehmen:</p><ul><li>Ernsthafte Zahlungsschwierigkeiten aufgrund ungünstiger wirtschaftlicher Verhältnisse, oder</li><li>wenn diese nach Einzug der Sozialversicherungsabgaben eintreten würden.</li></ul><p>Wichtig: Eine Stundung der Sozialbeiträge ist nur bei situativer Überschuldung möglich</p><p>Beantragung: Das Unternehmen muss den Antrag bei der zuständigen Krankenkasse stellen, bei der das Vorliegen der Voraussetzungen zu belegen ist.</p>",
     link: "https://www.ihk-muenchen.de/de/Service/Recht-und-Steuern/Arbeitsrecht/Bestehende-Arbeitsverh%C3%A4ltnisse-K%C3%BCndigung-und-Sozialversicherung/Corona-Virus-Dienstreisen-Arbeitsausfall-Arbeitsschutz/"
   }
-}
\ No newline at end of file
+}
+
+
+async function getDescriptions() {
+  descriptions.set(
+    await fetch(new URL(location.origin + '/api/descriptions'), { method: 'GET' }).then(res => res.json())
+  );
+}
+
+async function getSelectValues() {
+  const selects = await fetch(new URL(location.origin + '/api/selects'), { method: 'GET' })
+    .then(res => res.json())
+  selects.forEach(select => {
+    if(select.name === 'state') bundeslaender.set(select.options);
+    if(select.name === 'trade') gewerbe.set(select.options);
+    // Remove legal during rework
```

Comment:
couple of those comments in here. intentional?

Reply:
Yes, rework refers to general process rework which might take another week or two